### ISSUE TITLE
MAILER encryption/login params don't work

### ISSUE THREAD

**Issue by alexandru** [wallabag/docker#205]
I'm trying to self host Matomo and I've added these parameters to my environment as per #106:

```sh
SYMFONY__ENV__MAILER_TRANSPORT=smtp
SYMFONY__ENV__MAILER_HOST=smtp.fastmail.com
SYMFONY__ENV__MAILER_PORT=465
SYMFONY__ENV__MAILER_ENCRYPTION=ssl
SYMFONY__ENV__MAILER_AUTH_MODE=plain
SYMFONY__ENV__MAILER_USER=....
SYMFONY__ENV__MAILER_PASSWORD=....
```

Unfortunately they don't work. Debugging this I'm seeing these parameters should be seen by Ansible via `/etc/ansible/entrypoint.yml`, specified in `/etc/ansible/templates/parameters.yml.j2`, however the generated `/var/www/wallabag/app/config/parameters.yml` only contains these:

```
    mailer_transport: smtp
    mailer_user: ...
    mailer_password: ...
    mailer_host: smtp.fastmail.com
```

Am I missing anything? How can I get #106 to work?

**Comment by alexandru** [comment]
Investigating, I'm pretty sure that what happens is this:

1. `/var/www/wallabag/app/config/parameters.yml` does indeed get generated correctly from `/etc/ansible/templates/parameters.yml.j2`
2. `SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist` overwrites it

**Comment by alexandru** [comment]
I understand what happens.

That `composer install` makes `app/config/parameters.yml` to follow the template set by `app/config/parameters.yml.dist`.

And unfortunately those parameters are missing in `parameters.yml.dist` because `WALLABAG_VERSION=2.3.8` does not have them 🙁

**Comment by j0k3r** [comment]
Yep it'll be available in the 2.4.0 which isn't released yet
